ods-devenv/packer/build-ods-ami-ssm.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -eu
+set -o pipefail
+# build-ods-ami-ssm.sh builds an ODS image using Packer over SSM.
+# It depends on a prebuilt CentOS7 AMI which has SSM installed.
+# To use this in your AWS account, you must have an IAM instance profile setup
+# as described in https://docs.aws.amazon.com/systems-manager/latest/userguide/setup-instance-profile.html#instance-profile-add-permissions.
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+iam_instance_profile="SSMInstanceProfile"
```

Comment:
awesome! works like a charme.
Would be great if iam_instance_profile could be configured via arguments, too.